api/tests/unit/domain/models/user_test.js

```diff
@@ -95,4 +95,20 @@ describe('Unit | Domain | Models | User', () => {
 
   });
 
+  describe('#email', function() {
```

Comment:
ligne 8 : `should build an Organization from raw JSON` => `should build a User from raw JSON`

Reply:
OK